src/transforms/stack.d.ts

```diff
@@ -21,32 +21,110 @@ export type StackOrder =
   | any[]; // explicit ordinal values
 
 export interface StackOptions {
+  /**
+   * After all values have been stacked from zero, an optional **offset** can be
+   * applied to translate or scale the stacks:
+   *
+   * - null (default) - a zero baseline
+   * - *normalize* (or *expand*) - rescale each stack to fill [0, 1]
+   * - *center* (or *silhouette*) - align the centers of all stacks
+   * - *wiggle* - translate stacks to minimize apparent movement
+   * - a function to be passed a nested index, and start, end, and *z* values
+   *
+   * If a given stack has zero total value, the *expand* offset will not adjust
+   * the stack’s position. Both the *center* and *wiggle* offsets ensure that
+   * the lowest element across stacks starts at zero for better default axes.
+   * The *wiggle* offset is recommended for streamgraphs, and if used, changes
+   * the default order to *inside-out*.
+   */
   offset?: StackOffset | null;
+
+  /**
+   * The order in which stacks are layered:
+   *
+   * - null (default) - input order
+   * - *value* - ascending value order (or descending with **reverse**)
+   * - *x* for stackY, or *y* for stackX - alias of *value*
+   * - *sum* - order series by their total value
+   * - *appearance* - order series by the position of their maximum value
+   * - *inside-out* - order the earliest-appearing series on the inside
+   *   (recommended for the *wiggle* offset)
+   * - a named field or function of data - order data by priority
+   * - an array enumerating all the *z* values in the desired order
+   *
+   * The **reverse** option reverses the effective order. For the *value* order,
```

Comment:
Do we need to document the **reverse** option here? Isn’t it already covered below?

Reply:
👍 I've now removed the whole paragraph—the three sentences are all explicited elsewhere.